### DISCUSSION TITLE
Will changing the image size when training images affect the accuracy of inference, such as the option to maintain the aspect ratio of the image?

### DISCUSSION THREAD

**Discussion by zhuyuhang4** [ultralytics/ultralytics#15879] [General]
Because I want to input tensor directly during inference, I want to understand this problem

**Comment by glenn-jocher** [comment]
@zhuyuhang4 yes, changing the image size during training can affect inference accuracy. Maintaining the aspect ratio helps preserve the spatial relationships in the image, which can improve model performance. For more details, you can refer to our documentation on model training and inference.

**Reply by zhuyuhang4** [reply]
Thank you, is it important to maintain the aspect ratio for both training and inference?

**Reply by glenn-jocher** [reply]
Maintaining the aspect ratio during both training and inference is generally recommended as it helps preserve the spatial relationships in the image, which can improve model accuracy.
